README.md

```diff
@@ -434,226 +455,226 @@ bind::controls:
 
 - *Default*: '/etc/named/zone_lists'
 
-===
+---
+
+# Define Descriptions
+## Defined type `bind::acl`
 
-# Parameters for `define bind::acl`
+### Description
 
 Manage acl declarations.
 
 Must specify at least one of `entries` and `keys`.
 
-name (type: String)
-----
+### Parameters
+
+---
+#### name (type: String)
 Unique name of the `acl` declaration.
 
-entries (type: Array or undef)
 ---
+#### entries (type: Array or undef)
 List of entries for an `acl` declaration.
 
 - *Default*: undef
 
-keys (type: Array or undef)
 ---
+#### keys (type: Array or undef)
 List of keys for an `acl` declaration.
 
 - *Default*: undef
 
-===
+---
+
+## Defined type `bind::channel`
 
-# Parameters for `define bind::channel`
+### Description
 
 Manage a channel declaration. The types are fundamentally those of files or
-syslog, so one of `syslog_facility` and `files` must be populated.
+syslog, so one of `syslog_facility` and `file` must be populated.
+
+##### Example:
+```yaml
+bind::channels:
+  'my_syslog':
+    type: 'syslog'
+    syslog_facility: 'daemon'
+    severity: 'info'
+```
+---
+
+### Parameters
 
-name (type: String)
 ---
+#### name (type: String)
 Name of logging channel.
 
-type (type: String)
 ---
+#### type (type: String)
 Type of logging channel. Valid values are 'file', 'syslog', 'stderr' and 'null'.
 
 - *Required*
 
-file (type: String)
 ---
+#### file (type: String)
 Value of `file` option for channel. May be a relative path.
 
 - *Default*: undef
 
-severity (type: String)
 ---
+#### severity (type: String)
 Value of `severity` option for channel.
 
 - *Default*: undef
 
-syslog_facility (type: String)
 ---
+#### syslog_facility (type: String)
 Value of `syslog` option for channel.
 
 - *Default*: undef
 
-```
-# example
-
-bind::channels:
-  'my_syslog':
-    type: 'syslog'
-    syslog_facility: 'daemon'
-    severity: 'info'
-```
-===
+## Defined type `bind::key`
 
-# Parameters for `define bind::key`
+### Description
 
 Manage a key declaration.
 
-name (type: String)
+##### Example:
+```yaml
+bind::keys:
+  'key-external-transfer':
+    secret: 'generated_secret'
+  'key-internal-transfer':
+    secret: 'generated_secret'
+```
+---
+
+### Parameters
+
 ---
+#### name (type: String)
 Name of key.
 
-secret (type: String)
 ---
+#### secret (type: String)
 Value of `secret` option for key.
 
 - *Required*
 
-algorithm (type: String)
 ---
+#### algorithm (type: String)
 Value of `algorithm` option for key.
 
 - *Default*: 'hmac-md5'
 
-path (type: Integer)
 ---
+#### path (type: Integer)
 Absolute path to file containing the key declaration.
 
-- *Default*: "/etc/named/{name}.key"
-
-```
-# example
-bind::keys:
-  'key-external-transfer':
-    secret: 'generated_secret'
-  'key-internal-transfer':
-    secret: 'generated_secret'
-```
+- *Default*: "/etc/named/${name}.key"
 
-===
+## Defined type `bind::masters`
 
-# Parameters for `define bind::masters`
+### Description
 
 Manage a masters declaration.
 
-name (type: String)
```

Comment:
This moved to line 693 as I moved the examples to come before the parameter descriptions.